contracts/Interfaces/ILosslessGovernance.sol

```diff
@@ -36,6 +36,9 @@ interface ILssGovernance {
     function claimCommitteeReward(uint256 _reportId) external;
     function setCompensationAmount(uint256 _amount) external;
     function losslessClaim(uint256 _reportId) external;
+    function setRevShareAdmin(address _address) external;
```

Comment:
Maybe let's make share lowercase for consistency

Reply:
Addressed on new commits